src/components/DatePicker/DatePicker.jsx

```diff
@@ -166,6 +167,12 @@ export const DatePicker = ({
                     options,
                     event,
                 );
+            } else {
```

Comment:
looks good, could you please add a comment in the code about what this `else` block do